### PR TITLE
 remove library icon and add stack icon

### PR DESCRIPTION
**Pull request by mileskc**
BREAKING CHANGE: removes the library icon, which could be a breaking change but the library icon is
not currently used in the next repo

Also updates README for icons package

### CODE REVIEW

File: packages/icons/src/svgs/uncategorized/GoogleDrive.svg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" viewBox="0 0 32 32">
-    <path fill="currentColor" d="M8 26.573l4-6.667h16l-4 6.667H8z" class="GoogleDrive_svg_GoogleDrive_svg_path1"/>
-    <path fill="currentColor" d="M20 19.906h8L20 6.573h-8l8 13.333z" class="GoogleDrive_svg_GoogleDrive_svg_path2"/>
-    <path fill="currentColor" d="M4 19.906l4 6.667 8-13.333-4-6.667-8 13.333z" class="GoogleDrive_svg_GoogleDrive_svg_path3"/>
+    <path fill="currentColor" d="M8 26.573l4-6.667h16l-4 6.667H8z" class="GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_path1"/>
+    <path fill="currentColor" d="M20 19.906h8L20 6.573h-8l8 13.333z" class="GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_path2"/>
+    <path fill="currentColor" d="M4 19.906l4 6.667 8-13.333-4-6.667-8 13.333z" class="GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_GoogleDrive_svg_path3"/>
```
Comment (kylemh) [inline packages/icons/src/svgs/uncategorized/GoogleDrive.svg:4]: Could you revert these changes? Then it's perfect!

Comment (mileskc) [reply packages/icons/src/svgs/uncategorized/GoogleDrive.svg:4]: Yeah I'm not sure how that happened, I didn't even open that file - will do

Comment (kylemh) [reply packages/icons/src/svgs/uncategorized/GoogleDrive.svg]: I tried describing the problem in the updated README

**Review by kylemh — APPROVED** [review]

**Review by mileskc — COMMENTED** [review]
@kylemh is it ok to merge even though the UI test and review didn't go through? Or is that an indication of an issue

**Comment by kylemh** [comment]
Sorry for not seeing this. Approving for ya!
